packages/design-system/src/components/Autocomplete/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -8,6 +8,7 @@ exports[`Autocomplete renders Autocomplete component 1`] = `
     class="ds-u-clearfix"
   >
     <label
+      aria-hidden="true"
```

Comment:
I don't know where this is coming from. It doesn't seem to be happening in Storybook. This is something I'll need to fix before merging.